src/components/page-list.jsx

```diff
@@ -67,6 +63,19 @@ export default class PageList extends React.Component {
     );
   }
 
+  renderBody () {
+    if (this.props.pages) {
+      return this.props.pages.map(page => this.renderRow(page));
+    }
+    else {
+      return (
+        <tr>
+          <td colSpan={8}>You are not assigned to any pages.</td>
+        </tr>
+      );
+    }
+  }
+
```

Comment:
This works great and is easy 👍 

…but I wonder if it would be better to render this message *instead* of the table, rather than inside it. Not really sure if it makes the UI more or less clear, but it does mean we don’t need to change things here if we modify the number of columns. (Totally *not* a request for a change unless you agree. More of a passing thought.)

Reply:
After, a bit more diligence, I'm leaning towards taking the message out of the table. We will be modifying the number of columns in #168. We'll also probably add more columns once we put annotations back in.

But I like having the empty table with header, only because a small message by itself in wide open water scares me (just my own aesthetic). So maybe place the message underneath the header?